prow/config/tide.go

```diff
@@ -132,6 +132,24 @@ type Tide struct {
 	// combined status; otherwise it may apply the branch protection setting or let user
 	// define their own options in case branch protection is not used.
 	ContextOptions TideContextPolicyOptions `json:"context_options,omitempty"`
+
+	// BatchSizeLimitMap is a key/value pair of an org or org/repo as the key and
+	// integer batch size limit as the value. The empty string key can be used as
```

Comment:
Why not use `"*"` for default like we do everywhere else?

Reply:
I didn't realize that was more common. I'll switch it.